### ISSUE TITLE
is_training=True

### ISSUE THREAD

**Issue by ilkarman** [pudae/tensorflow-densenet#11]
Hi, I was curious why it's the case for this implementation (and also slim's resnet_v1) that fine-tuning a model with is_training=True and then running inference with is_training=True gives better results than inference with is_training=False?

I searched and this has been mentioned here: https://github.com/tensorflow/models/issues/1288 & https://github.com/tensorflow/models/pull/2138#issuecomment-323225293 & https://github.com/tensorflow/models/issues/391

And the slim [walkthrough](https://github.com/tensorflow/models/blob/master/research/slim/slim_walkthrough.ipynb) does the same thing in the last cell (inference with is_training=True).

**Comment by pudae** [comment]
Do you use dropout? I found bug in my code. 

this code.. (nets/densenet.py)
```
if dropout_rate:
  net = tf.nn.dropout(net)
```
must be like this.
```
if dropout_rate:
  keep_prob = 1.0 - dropout_rate
  net = slim.dropout(net, keep_prob=keep_prob)
```

I'm not fixed it yet

**Comment by BowieHsu** [comment]
@ilkarman I meet the same problem, for now i have to set is_training=True at test phase which seems like a ugly solution, i think the reason should be slim code, when set is_training=False, the graph can not find moving mean and var, so the results are really bad.
